tests/CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ add_test(unit_tests            ${TEST_WRAP} unit_tests ${CMAKE_CURRENT_SOURCE_DI
 add_test(router_engine_test    ${TEST_WRAP} unit2 -v router_engine_test)
 add_test(management_test       ${TEST_WRAP} unit2 -v management)
 add_test(router_policy_test    ${TEST_WRAP} unit2 -v router_policy_test)
```

Comment:
BTW: this has changed on master: instead of unit2 we've parametrized this.  See master for latest changes.